doc/user_guide/transform.rst

```diff
@@ -699,6 +699,38 @@ last_value    None       Assigns a value from the last data object in the curren
 nth_value     Number     Assigns a value from the nth data object in the current sliding window frame. If no such object exists, assigns ``null``. Requires a non-negative integer parameter that indicates the offset from the start of the window frame. This operation must have a corresponding entry in the `fields` parameter array.
 ============  =========  =========================================================================================================================================================================================================================================================================================================================
 
+Notice that when we use an aggregation function within an aggregate transform, the data objects are collapsed.
+However, when we use an aggregation function within a window transform, the data objects are augmented.
+It's by augmenting, rather than collapsing, that we can compute statistics, such as `z-scores`_.
+For example, consider the following time series of stock prices:
```

Comment:
I'm not entirely clear what is meant by "collapsed" and "augmented" in this context.

Reply:
Yes, good question. What I want to communicate is that an aggregate transform with an aggregation function is like an SQL `GROUP BY` but a window transform with an aggregation function is like broadcasting in Pandas. In the latter case, each data object has a new property (is augmented with), however, there are the same number of data objects. In the former case, there are (probably) a different number of objects.

Reply:
Ah, maybe something like "While aggregate transforms compute a single value summarizing the dataset, window transforms effectively add a new column to the dataset, constructed from local aggregates at each row."

Reply:
I'm not sure what you mean by "local aggregates around each value in the column". Local to the data object, which is synonymous with adding a new column? I updated this section in a previous commit, but Github's UI unfortunately isn't displaying it here. Would you mind taking a look? I'll comment it now.

Reply:
By "local aggregates" I was trying to say "windowed aggregates" without being so tautological :smile:

Reply:
It's local in the sense that the values nearby the point are aggregated and put in the new column.

Reply:
Ah, I see! I've replaced the section with the following:

> While an aggregate transform computes a single value that summarises all data objects, a window transform adds a new property to each data object. This new property is computed from the neighbouring data objects: that is, from the data objects delimited by the window field definition.

I felt that "neighboring" was more descriptive than "local", but I'm happy to change it. Hopefully the reference to the window field definition isn't too tautological.

Reply:
Looks good, thanks!